File: Session1-view/session1/session1/pojo/OrderItems.cs
using Microsoft.SqlServer.Server;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace session1.pojo
{
    internal class OrderItems
    {
        private string OId;
        private Orders Order;
        private Parts Parts;
        private string BatchNumber;
        private string Amount;

        public OrderItems()
        {
        }

        public OrderItems(string oId, Orders order, Parts parts, string batchNumber, string amount)
        {
            OId = oId;
            Order = order;
            Parts = parts;
            BatchNumber = batchNumber;
            Amount = amount;
        }

        public string OId1 { get => OId; set => OId = value; }
        public string BatchNumber1 { get => BatchNumber; set => BatchNumber = value; }
        public string Amount1 { get => Amount; set => Amount = value; }
        internal Orders Order1 { get => Order; set => Order = value; }
        internal Parts Parts1 { get => Parts; set => Parts = value; }


    }
}

File: Session1-service/src/main/java/com/yang/session/Pojo/OrderItems.java
package com.yang.session.Pojo;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import org.springframework.stereotype.Repository;

@Data
@AllArgsConstructor
@NoArgsConstructor
@Repository
public class OrderItems {
    private long OId;
    private Orders Order;
    private Parts Part;
    private String BatchNumber;
    private double Amount;
}
